prime-router/src/main/kotlin/tokens/SenderUtils.kt

```diff
@@ -39,7 +37,12 @@ class SenderUtils {
             return jws
         }
 
```

Comment:
Note:   SenderUtils.kt is a suite of little functions that are useful when you want to write kotlin code that pretends to be someone being a Sender to ReportStream.   Useful for testing, and for creating little example programs.

I had to make some minor tweaks to the methods here so that I could send parameters in URL or in the body.